plugins/services/src/js/components/forms/MultiContainerVolumesFormSection.js

```diff
@@ -74,6 +76,26 @@ class MultiContainerVolumesFormSection extends Component {
     });
   }
 
+  getUnknownVolumeConfig(volumes, key, offset) {
+    return (
+      <MountService.Mount
+        type="CreateService:MultiContainerVolumes:UnknownVolumes"
+        volumes={volumes}
+        index={key}
+        offset={offset}
+        data={this.props.data}
+        errors={this.props.errors}
+      >
+        <FieldLabel>
+          Unable to edit this Volume{" "}
```

Comment:
Why do we need the empty space at the end?

Reply:
I still believe we don't need this extra trailing space

Reply:
You are right. At least I do not see any more an issue there...
removed the space